pkg/build/build.go

```diff
@@ -514,6 +514,10 @@ func New(sigh context.Context, opts ...Option) (*Context, error) {
 
 	// SOURCE_DATE_EPOCH will always overwrite the build flag
 	if v, ok := os.LookupEnv("SOURCE_DATE_EPOCH"); ok {
+		if v == "" {
+			ctx.Logger.Warnf("SOURCE_DATE_EPOCH is specified but empty, setting it to 0")
+			v = "0"
```

Comment:
should this be `$(date +%s)` instead?

Reply:
Weak preference for 0 for reproducibility (thus cache hits) during local dev.